src/common/initializable-state/create.ts

```diff
@@ -5,11 +5,13 @@
 
 import type { DiContainerForInjection, Injectable, InjectionToken } from "@ogre-tools/injectable";
 import { getInjectable } from "@ogre-tools/injectable";
+import type { Runnable } from "../runnable/run-many-for";
 
 export interface CreateInitializableStateArgs<T> {
   id: string;
   init: (di: DiContainerForInjection) => Promise<T> | T;
   injectionToken?: InjectionToken<InitializableState<T>, void>;
+  when: InjectionToken<Runnable<void>, void>;
```

Comment:
Nit: `void` should be default value in both type parameters using it.

Reply:
This is also seen in many other places in the PR.

Reply:
Yes, I prefer just using it here so that I don't have to go searching for a different file to figure out what the type param is.